runtime/src/xcm_config.rs

```diff
@@ -116,31 +118,30 @@ pub struct DenyReserveTransferToRelayChain;
 impl ShouldExecute for DenyReserveTransferToRelayChain {
 	fn should_execute<RuntimeCall>(
 		origin: &MultiLocation,
-
-		message: &mut Xcm<RuntimeCall>,
-		_max_weight: XCMWeight,
-		_weight_credit: &mut XCMWeight,
+		message: &mut [Instruction<RuntimeCall>],
+		_max_weight: Weight,
+		_weight_credit: &mut Weight,
 	) -> Result<(), ()> {
-		if message.0.iter().any(|inst| {
+		if message.iter().any(|inst| {
```

Comment:
Looks like this is not up-to-date with Parachain template v0.9.40 code (see [here](https://github.com/substrate-developer-hub/substrate-parachain-template/pull/153/files#diff-7fbdadf7a2bc29be480bf3b857bacfdabc819f248e848752df2af46e19d8b32eL123-L152)).

Reply:
Thanks for catching this.